src/schemas/json/github-workflow.json

```diff
@@ -1177,10 +1224,7 @@
                         }
                       ]
                     }
-                  ],
-                  "additionalItems": {
-                    "type": "string"
-                  }
+                  ]
```

Comment:
This might have fixed the validator, but this is technically incorrect. These strings can be anything. Self-hosted runners can have custom labels that (afaict) don't have any limitations

Reply:
selft-hosted runners list of strings can be anything but have only one limitation. That it must start with `self-hosted`
Or this rule is wrong. Let see what this new external PR will bring. https://github.com/github/docs/pull/9387

There is also a [JSON ](https://github.com/SchemaStore/schemastore/blob/master/src/test/github-workflow/runs-on.json)test file for this situation.

https://github.com/SchemaStore/schemastore/blob/45a741de615b92152528505cac6bdbac9050b148/src/test/github-workflow/runs-on.json#L140-L147

What is missing in this part of the the schema is the best practice method.

- Add "minItems" for each item inside anyof
- Add  "additionalItems": { "type": "string"} for each item inside anyof

At this moment any potential "additionalItems" is undefined.